app/views/posts/index.html.erb

```diff
@@ -23,5 +23,8 @@
         </div>
       </li>
     <% end %>
+    <% if @posts.empty? %>
```

Comment:
I think we automatically load all posts, so I guess it would be only in the case that there are no posts (like now :P)